transport/ProbeServer.cpp

```diff
@@ -274,11 +281,40 @@ void ProbeServer::replyStunOk(Endpoint& endpoint, const SocketAddress& destinati
         response.addFingerprint();
 
         endpoint.sendStunTo(destination, response.header.transactionId.get(), &response, response.size(), timestamp);
+        return true;
     }
-    else
+
+    if (endpoint.getTransportType() != ice::TransportType::UDP)
```

Comment:
I believe this is another bug. If we received something that was not properly authenticated, we would close UdpEndpoint which is shared port 10000